examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,83 @@
+#The software contained in this file is free and unencumbered software released into the public domain.
```

Comment:
What is this? Use the copyright headers in the rest of the project

Reply:
We want anyone to be able to incorporate any of the code in this file into their proprietary application without restriction.  This is the "Public Domain" header.